command/datastore.go

```diff
@@ -25,10 +29,18 @@ const (
 	updateBatchSize = 500000
 )
 
-func createDatastore(ctx context.Context, dir, dsType string, rmExisting bool) (datastore.Batching, string, error) {
-	if dsType != "levelds" {
-		return nil, "", fmt.Errorf("only levelds datastore type supported, %q not supported", dsType)
+func createDatastore(ctx context.Context, dirOrTable, dsType, region string, rmExisting bool) (datastore.Batching, string, error) {
```

Comment:
Suggestion: pass a DB specific config object here so you don't have to add params that aren't relevant to other types of datastore.

Reply:
good point, will do 👍🏻

Reply:
done!